src/hooks/loadables/useLoadBalances.ts

```diff
@@ -1,23 +1,33 @@
-import { useEffect } from 'react'
+import { useEffect, useMemo } from 'react'
 import { getBalances, type SafeBalanceResponse } from '@safe-global/safe-gateway-typescript-sdk'
 import { useAppSelector } from '@/store'
 import useAsync, { type AsyncResult } from '../useAsync'
 import { Errors, logError } from '@/services/exceptions'
-import { selectCurrency } from '@/store/settingsSlice'
+import { selectCurrency, selectSettings } from '@/store/settingsSlice'
 import { selectSafeInfo } from '@/store/safeInfoSlice'
+import { useCurrentChain } from '../useChains'
+import { FEATURES, hasFeature } from '@/utils/chains'
 
 export const useLoadBalances = (): AsyncResult<SafeBalanceResponse> => {
   // use the selector directly because useSafeInfo is memoized
   const { data: safe } = useAppSelector(selectSafeInfo)
   const currency = useAppSelector(selectCurrency)
+  const settings = useAppSelector(selectSettings)
+  const chain = useCurrentChain()
+  const isDefaultTokenList = useMemo(() => {
+    const hasDefaultList = chain !== undefined && hasFeature(chain, FEATURES.DEFAULT_TOKENLIST)
+    return hasDefaultList && (!settings.tokenList || settings.tokenList === 'DEFAULT')
+  }, [chain, settings.tokenList])
 
   // Re-fetch assets when the entire SafeInfo updates
   const [data, error, loading] = useAsync<SafeBalanceResponse | undefined>(
     async () => {
       if (!safe) return
-      return getBalances(safe.chainId, safe.address.value, currency)
+      return getBalances(safe.chainId, safe.address.value, currency, {
+        trusted: isDefaultTokenList,
```

Comment:
I suggest we use the word `trusted` instead of `default` in our variable naming. In the UI it may be called "default", but I think logically trusted is more understandable and is consistent with this API param.